app/code/Magento/Sales/Block/Adminhtml/Order/View.php

```diff
@@ -451,7 +454,7 @@ public function getBackUrl()
             return $this->getOrder()->getBackUrl();
         }
 
-        return $this->getUrl('sales/*/');
+        return $this->getUrl('sales/*/', ['action' => 'index']);
```

Comment:
I would suggest a direct way here instead of using workarounds. For example, we can simply invoke
```
$this->_urlBuilder->getUrl('sales/*/');
```
without calling `$this->getUrl()`. In that way, we get rid of the order id and avoid additional unnecessary conditions in the `getUrl()` method of the block.

Reply:
Hello @rogyar ,
Thanks for the review, will make suggested changes and update PR.